week3/webApplicationHw/Controllers/CarController.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.AspNetCore.Mvc;
+using webApplicationHw.Models;
+
+namespace webApplicationHw.Controllers;
+
+
+[ApiController]
+[Route("[controller]")]
+public class CarController : ControllerBase
+{
+
+    public static List<Car> Cars { get; set; } = new List<Car>(){
+        new Car(){ ID=0, Brand="Fiat", Model="500", Year=2019},
```

Comment:
Use the `Shift + Alt + F` shortcut to auto format the code, makes everything more easily readable